apps/fault/src/main.c

```diff
@@ -168,25 +168,28 @@ set_good_magic_and_set_pc(seL4_CPtr tcb, seL4_Word new_pc)
     assert(!error);
 }
 
-static void measure_vm_fault_fn(int argc, char **argv) {
-    assert (argc == N_FAULTER_ARGS);
```

Comment:
If the changes in this commit are only changing code introduced in the previous commit, then it would be easier to review if they were merged back into the original commit.